app/controllers/set_ups_controller.rb

```diff
@@ -34,4 +35,8 @@ def set_up_params
   def set_set_up
     @set_up = SetUp.find(params[:id])
   end
+
+  def check_admin
+    redirect_to health_checks_path unless current_user.admin?
```

Comment:
Do we need to have a modal or flash to tell employee that they cannot access the set_up?